homeassistant/components/brother/sensor.py

```diff
@@ -60,18 +60,17 @@ def __init__(
     ) -> None:
         """Initialize."""
         super().__init__(coordinator)
-        self._description = SENSOR_TYPES[kind]
-        self._name = f"{coordinator.data.model} {self._description[ATTR_LABEL]}"
-        self._unique_id = f"{coordinator.data.serial.lower()}_{kind}"
-        self._device_info = device_info
-        self.kind = kind
+        description = SENSOR_TYPES[kind]
         self._attrs: dict[str, Any] = {}
-        self._attr_state_class = self._description[ATTR_STATE_CLASS]
-
-    @property
-    def name(self) -> str:
-        """Return the name."""
-        return self._name
+        self._attr_device_class = description[ATTR_DEVICE_CLASS]
```

Comment:
```suggestion
        self._attr_device_class = description.get(ATTR_DEVICE_CLASS)
```

If you do this, you can remove all the None files from the dict in the const file

Reply:
Good idea!